ui/app/components/ui/unit-input/index.scss

```diff
@@ -48,14 +49,14 @@
 
   &__input-container {
     display: flex;
-    align-items: flex-start;
+    align-items: center;
     padding-bottom: 4px;
```

Comment:
I had suggested in the last PR that this be reduced to `2px`, to more closely match the old height of this input (https://github.com/MetaMask/metamask-extension/pull/9074/files#r466403886)

I think that suggestion still has merit? :thinking: The difference is fairly small though.

Reply:
Done :D